storm-client/src/jvm/org/apache/storm/security/auth/authorizer/SimpleACLAuthorizer.java

```diff
@@ -40,6 +40,11 @@
         "fileUpload",
         "getNimbusConf",
         "getClusterInfo",
+        "getLeader",
+        "isTopologyNameAllowed",
+        "getTopologySummaries",
+        "getTopologySummaryByName",
+        "getTopologySummary",
```

Comment:
getTopologyInfoByName and getTopologyInfoByNameWithOpts need here?

Reply:
All variations for for _getTopologyInfo_ method are using _getTopologyInfo_ for now. I do not intend to change implementation _getTopologyInfoWithOptsImpl_ at this point to refactor checkAuthorization else where.